data/kush6291.json

```
@@ -0,0 +1,17 @@
+{
+"name":"Kushagra Chavan (Test)",
+  "displayStatsPublic": true,
+  "type": "Personal",
+  "bio" "I am a DevOps Engineer",
+  "avatar": "https://github.com/kush6291.png"
+  "links": [
+      {
+        "name": "Follow me on twitter"
```

```suggestion
        "name": "Follow me on twitter",
```
Please add comma here